scipy/signal/signaltools.py

```diff
@@ -1826,6 +1826,14 @@ def medfilt2d(input, kernel_size=3):
     efficient implementation of a median filter and therefore runs much faster.
     """
     image = np.asarray(input)
+
+    # see sigtools_median2d in sigtoolsmodule.c for supported types
+    # for sigtools._medfilt2d
```

Comment:
Maybe better to spell it out here, that code isn't exactly clear.

Reply:
Will do.